jujugui/static/gui/src/app/components/string-config/string-config.js

```diff
@@ -30,6 +30,11 @@ YUI.add('string-config', function() {
       this.setState({ value: nextProps.config });
     },
 
+    shouldComponentUpdate: function(nextProps, nextState) {
+      return nextState.value !==
+        this.getDOMNode().querySelector('.string-config--value').innerText;
```

Comment:
I think `getDOMNode` is deprecated, I think you want `ReactDOM.findDOMNode(this)`, or whatever the correct new syntax is.